cw/simple_capture_aes_sca.py

```diff
@@ -212,17 +230,26 @@ def abort_handler_during_loop(this_project, sig, frame):
 
                 # Sanity check retrieved data (wave)
                 assert len(waves[i, :]) >= 1
-                # Create CW Trace
-                trace = cw.Trace(waves[i, :], text, ciphertext, key)
+                # Add trace.
+                if OTTraceLib:
+                    # OT Trace Library
+                    trace = Trace(wave=waves[i, :].tobytes(),
+                                  plaintext=text,
+                                  ciphertext=ciphertext,
+                                  key=key)
+                    project.write_to_buffer(trace)
+                else:
+                    # CW Trace
+                    trace = cw.Trace(waves[i, :], text, ciphertext, key)
+                    # Append trace Library trace to database
+                    # FIXME Also use uint16 as dtype for 8 bit WaveRunner for
```

Comment:
Please change the FIXME to a TODO because thats what we use in the OpenTitan project.